README.md

```diff
@@ -150,16 +150,28 @@ using Microsoft.FeatureManagement;
 
 public class Startup
 {
-  public void ConfigureServices(IServiceCollection services)
-  {
-      services.AddFeatureManagement();
-  }
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddFeatureManagement();
+    }
 }
 ```
 
-This tells the feature manager to use the "FeatureManagement" section from the configuration for feature flag settings.
+By default, the feature manager retrieves feature flag configuration from the "FeatureManagement" section of the .NET Core configuration data. If the "FeatureManagement" section does not exist, the configuration will be considered empty.
 
-**Advanced:** The feature manager looks for feature definitions in a configuration section named "FeatureManagement". If the "FeatureManagement" section does not exist, the configuration will be considered empty.
+You can also specify that feature flag configuration should be retrieved from a different configuration section by calling configuration.GetSection and passing in the name of the desired section. The following example tells the feature manager to read from a different section called "MyFeatureFlags" instead:
+``` C#
+services.AddFeatureManagement(configuration.GetSection("MyFeatureFlags"));
+```
+
+### Scoped Feature Management Services
+By default, the feature manager and feature filters should be registered as singleton services by calling `AddFeatureManagement`.
+
+There are scenarios that feature filters are not necessarily be singleton. For example, users may want to use feature filters which consume scoped services for context information. In this case, the feature manager and feature filters should be registered as scoped services by calling `AddScopedFeatureManagement`:

```

Comment:
```suggestion
The `AddFeatureManagement` method adds feature management services as singletons within the application, but there are scenarios where it may be necessary for feature management services to be added as scoped services instead. For example, users may want to use feature filters which consume scoped services for context information. In this case, the feature manager and feature filters should be registered as scoped services by calling `AddScopedFeatureManagement`:
```